### PR TITLE
[TG-134] Added the ability to remove devices

### PR DESCRIPTION
**Pull request by Todmy**
Added the ability to remove devices. Also made little changes in folder structure.

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/garage-it/SmartHouse-frontend/pull/32" target="_blank">ReviewNinja

File: src/app/sensors/list/device-list.scss [hunk 1]
```diff
@@ -33,6 +34,12 @@
     }
 }
 
+table {
+    button[name=remove-device] {
+        padding: 0 15px;
+    }
+}
```
Comment (quasiunafantasia) [inline src/app/sensors/list/device-list.scss:51]: we shouldn`t use tag selectors in styles, you can find more info in [markup styleguide](https://github.com/garage-it/SmartHouse-frontend/wiki/HTML-style-guide). dont know if it was ever used though

**Comment by Todmy** [comment]
@quasiunafantasia FIXED

File: src/app/sensors/details/sensor-detail.service.js [hunk 2]
```diff
@@ -8,7 +8,9 @@ export default class SensorDetailService {
     }
 
     get(id) {
-        return this.http.get(`/sensors/${id}`);
+        const sensorId = id || '';
```
Comment (artem713) [inline src/app/sensors/details/sensor-detail.service.js]: You can use default params
`get(id = '') {`

**Comment by Todmy** [comment]
@artem713 FIXED

**Comment by kosich** [comment]
⭐

**Pull request merged**